app/views/admin/r_and_r_items/show.html.erb

```diff
@@ -0,0 +1,152 @@
+<div class="mb-2">
+  <h1>R&R request: <%= @admin_r_and_r_item.title %> </h1>
+</div>
+
+
+<div class="row mb-2">
+  <div class="col">
+    <%= link_to 'Back to R & R list', admin_r_and_r_items_path, class: "btn btn-primary" %>
+  </div>
+</div>
+
+<div class="row mb-2">
+  <div class="col">
+    <%= link_to 'Edit this R & R request', admin_edit_r_and_r_item_path(@admin_r_and_r_item), class: "btn btn-primary" %>
+  </div>
+</div>
+
+
+<% if @admin_r_and_r_item.ready_to_move_to_digitization_queue %>
+  <div class="row mb-2">
+    <div class="col">
+      <%= link_to "Send to Digitization Queue", new_admin_digitization_queue_items_path(r_and_r_item: @admin_r_and_r_item.id, collecting_area: @admin_r_and_r_item.collecting_area), class: "btn btn-primary" %>
+    </div>
+  </div>
+<% end %>
```

Comment:
1. Should we try to enforce `ready_to_move_to_digitization_queue?`, or just let the user do it when appropriate

2. If we are enforcing it, guarding the display in the view is not enough -- the controller also has to refuse to do it if it's not allowed

Reply:
Because I share your fear of one-to-one relationships, I've made some simple changes, and the relationship is many-to-many. You can now create more than one DQ item from an R&R item.